tts-python-deploy.ipynb

```diff
@@ -1,48 +1,83 @@
+    "\n",
+    "### Riva-build\n",
+    "\n",
+    "This step helps build a Riva-ready version of the model. It’s only output is an intermediate format (called a Riva Model Intermediate Representation (`.rmir`)) of an end-to-end pipeline for the supported services within Riva. Let’s consider two TTS models:\n",
+    "\n",
+    "* [FastPitch](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa) (spectrogram generator)\n",
+    "* [HiFi-GAN](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa) (vocoder).<br>\n",
+    "\n",
+    "`riva-build` is responsible for the combination of one or more exported models (`.riva` files) into a single file\n",
+    "containing an intermediate format called `.rmir`. This file contains a\n",
+    "deployment-agnostic specification of the whole end-to-end pipeline along with all the assets required for the\n",
+    "final deployment and inference. Refer to the [Riva documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/tts/tts-custom.html#fastpitch-and-hifi-gan) for more information.\n",
+    "\n",
+    "### Riva-deploy\n",
+    "\n",
+    "The deployment tool takes as input one or more `.rmir` files and a target model repository directory. It creates an ensemble configuration specifying the pipeline for\n",
+    "the execution and finally writes all those assets to the output model repository directory."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "5106ee0d",
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "19d3303d",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "import pathlib\n",
-    "import warnings"
+    "---\n",
+    "### Set configs and params.\n",
+    "Please update the parameters in the following code block:\n",
+    "- `machine_type`: Type of machine the tutorial is being run on. Acceptable values are `AMD64`, `ARM64_linux`, `ARM64_l4t`. Defaults to `AMD64`.  \n",
+    "- `target_machine`: Type of machine the RMIR will be deployed on. Acceptable values are `AMD64`, `ARM64_linux`, `ARM64_l4t`. Defaults to `AMD64`.  \n",
+    "- `acoustic_model`: Full path for acoustic_model `.riva` file. Defaults to `$pwd/speechsynthesis_en_us_fastpitch_ipa_vdeployable_v1.0/FastPitch_44k_EnglishUS_IPA.riva` which is the default download path for the NGC example model. This can be replaced with a custom acoustic model `.riva` checkpoint.  \n",
+    "- `vocoder`: Full path for vocoder `.riva` file file. Defaults to `$pwd/speechsynthesis_en_us_hifigan_ipa_vdeployable_v1.0/HifiGAN_44k_EnglishUS_IPA.riva` which is the default download path for the NGC example model. This can be replaced with a custom vocoder .riva checkpoint.  \n",
```

Comment:
```suggestion
    "- `vocoder`: Full path for vocoder `.riva` file. Defaults to `$pwd/speechsynthesis_en_us_hifigan_ipa_vdeployable_v1.0/HifiGAN_44k_EnglishUS_IPA.riva` which is the default download path for the NGC example model. This can be replaced with a custom vocoder `.riva` checkpoint.  \n",
```